.github/workflows/create-release.yml

```
@@ -0,0 +1,34 @@
+name: Create Release (Versioned)
+
+on:
+  workflow_run:
+      workflows: ["Publish NPM Package"]
+      types:
+        - completed
+
+jobs:
+  create-release:
+    # only run if the "Publish NPM Package" workflow was successful
```

Nit. All other comments are capitalized, I'd prefer if this one was too.
```suggestion
    # Only run if the "Publish NPM Package" workflow was successful
```